src/utils/useGetAddressFromLatLng.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-underscore-dangle,camelcase */
+import { useEffect, useState } from 'react'
+import { LatLng } from 'leaflet'
+import proj4, { InterfaceCoordinates } from 'proj4'
+import { ENDPOINTS } from '../../example/config'
+import { CRS_CONFIG } from './getCrsRd'
+
+type Links = {
+  self: {
+    href: string
+  }
+  next: {
+    href: string
+  }
+  previous: {
+    href: string
+  }
+}
+
+type VboStatus = { code: string; omschrijving: string }
+
+type Address = {
+  _links: Links
+  _display: string
+  landelijk_id: string
+  hoofdadres: boolean
+  vbo_status: VboStatus
+  dataset: string // Todo: could be an enum?
+}
+
+type ValidResponse = {
+  _links: Links
+  count: number
+  results: Address[]
+}
+
+function requestFormatter(baseUrl: string, xy: InterfaceCoordinates) {
+  proj4.defs(CRS_CONFIG.RD.code, CRS_CONFIG.RD.projection)
+
+  const transformCoords = proj4(CRS_CONFIG.WGS84.code, CRS_CONFIG.RD.code)
+
+  const { x, y } = transformCoords.forward(xy)
+  return `${baseUrl}${x},${y},50`
+}
+
+const useGetAddressFromLatLng = () => {
+  const [addresses, setAddresses] = useState<ValidResponse>()
+  const [latLng, setLatLng] = useState<LatLng>()
+  const [loading, setLoading] = useState(false)
+
+  useEffect(() => {
+    if (latLng) {
+      setLoading(true)
+      const xy: InterfaceCoordinates = {
+        x: latLng.lng,
+        y: latLng.lat,
+      }
+
+      window
```

Comment:
Don't think we need to use the window here, we can call `fetch` directly from the global namespace.

Reply:
Will update eslint 👍